ssg/constants.py

```diff
@@ -456,6 +456,8 @@
     "yum": "cpe:/a:yum",
     "zipl": "cpe:/a:zipl",
     "net-snmp": "cpe:/a:net-snmp",
+    "uefi": "cpe:/a:uefi",
+    "bios": "cpe:/a:bios",
```

Comment:
These will actually trip the Bash and Ansible platform checks.
The remediation platforms assume the non-machine platforms are "package CPEs".
For example for platform `sssd-ldap`:
`if rpm --quiet -q sssd-ldap &amp;&amp; [ ! -f /.dockerenv ] &amp;&amp; [ ! -f /run/.containerenv ]; then`

Reply:
I see. Which one should I use? `"cpe:/o:bios"` ?

Reply:
I've updated to `"cpe:/o:bios"` and `"cpe:/o:uefi"`

Reply:
I reverted back as you probably noticed.